components/admin_console/admin_sidebar/admin_sidebar.jsx

```diff
@@ -343,7 +343,9 @@ export default class AdminSidebar extends React.Component {
 
         const customPlugins = [];
         if (this.props.config.PluginSettings.Enable) {
-            Object.values(this.props.plugins).forEach((p) => {
+            Object.values(this.props.plugins).sort((a, b) => {
+                return (a.name > b.name) ? 1 : -1;
```

Comment:
Minor issue, but this sort comparison isn't stable in the presence of duplicate `name` fields. This itself is relatively unlikely, but it should be easy to fix by comparing the `name` and then `id` with something like:

```js
Object.values(this.props.plugins).sort((a, b) => {
    const nameCompare = a.name.localeCompare(b.name);
    if (nameCompare !== 0) {
        return nameCompare;
    }

    return a.id.localeCompare(b.id);
});
```